lib/element-helper-syntax-plugin.js

```diff
@@ -31,37 +31,18 @@ module.exports = class ElementHelperSyntaxPlugin {
 
       MustacheStatement(node) {
         if (isElementHelper(node, locals)) {
-          throw error('cannot be appended to the DOM directly', node);
+          const { path, params, hash } = transformParts(node, b);
+
+          return b.mustache(path, params, hash, false, node.loc);
```

Comment:
Intead of returning `transformParts()` as object, they could also be returned as array and then:

```ts
return b.mustache(...[...transformParts(node, b), false, node.loc]);
```

yet this didn't provide the best readability to me.